ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template4.kt

```diff
@@ -366,12 +317,15 @@ private fun ColumnScope.OfferName(
 }
 
 @Composable
-private fun CheckmarkBox(isSelected: Boolean, colors: TemplateConfiguration.Colors) {
+private fun CheckmarkBox(
+    isSelected: Boolean,
+    colors: TemplateConfiguration.Colors,
+    modifier: Modifier = Modifier,
+) {
     Box(
-        modifier = Modifier
+        modifier = modifier
             .size(Template4UIConstants.checkmarkSize)
-            .clip(CircleShape)
-            .background(colors.accent2.copy(alpha = Template4UIConstants.fadedColorOpacity)),
```

Comment:
@NachoSoto I removed this, I didn't really see any difference 🤔

Reply:
I think this was dead code from template 2